src/global/base.scss

```diff
@@ -0,0 +1,6 @@
+@font-face {
+  src: url("/docs/assets/fonts/eina-700.woff2") format("woff2"), url("/docs/assets/fonts/eina-700.woff") format("woff");
```

Comment:
Do we not have the Eina font in the shared components? If not we should probably this there